docs/build.rb

```diff
@@ -0,0 +1,1143 @@
+require 'fileutils'
+require 'shellwords'
```

Comment:
Do we still need this given the one line that was using it has been commented out?